wtxmgr/query.go

```diff
@@ -517,12 +517,24 @@ func (s *Store) PreviousPkScripts(rec *TxRecord, block *Block) ([][]byte, error)
 					// unmined transaction before including
 					// the output script.
 					k := canonicalOutPoint(&prevOut.Hash, prevOut.Index)
-					if existsRawUnminedCredit(ns, k) == nil {
+					vUC := existsRawUnminedCredit(ns, k)
+					if vUC == nil {
 						continue
 					}
 
-					pkScript, err := fetchRawTxRecordPkScript(
-						prevOut.Hash[:], v, prevOut.Index)
+					// If we encounter an error here, it likely means
+					// we have a legacy outpoint. Ignore the error and
```

Comment:
Oh, I see you, mean the second error. If the first one succeeds the second one should not fail. I can add error handling there.